sites/gcweb-menu/_base.scss

```diff
@@ -3,7 +3,7 @@
  */
 
 /* Special style for the home page - GCWeb v5 (gcweb-menu) */
-.home {
+.home  + .gcweb-menu {
 	.gcweb-menu {
```

Comment:
There is no instance where a "gcweb-menu" are inside another "gcweb-menu".

If this is for enable a smooth transition for our home page, please add a comment.